csuntechlab/iSTART
# STR-131 -- FE: Create card-flip template

# Description
Created just the template of the card-flip and used test data for testing the functionality and looks.
  
# Testing
1. Login as either `cyndee.bellamy` or `april.feldman`.
2. Click on any substance to visit the module view.
3. Ensure the `cardflip` template appears and is mobile responsive.
4. Make sure it meets the requirements of both the mock-up and alcohol slide.
